joint_trajectory_controller:
  ros__parameters:
    joints: 
    - joint_a1
    - joint_a2
    - joint_a3
    - joint_a4
    - joint_a5
    - joint_a6

    command_interfaces:
      - position
    state_interfaces:
      - position
    state_publish_rate: 250.0
    action_monitor_rate: 20.0
